upgrade/pack_builder_instantiations/UpgradePackBuilderFacade.cpp

```diff
@@ -51,12 +59,42 @@ namespace main_
         builder.WriteUpgradePack(outputFilename, fileSystem);
     }
 
+    infra::Optional<uint8_t> UpgradePackBuilderFacade::GetOrder(const std::string& targetName, const std::map<uint8_t, std::vector<std::string>>& orderedTargets) const
+    {
+        for (const auto& targets : orderedTargets)
+        {
+            const auto targetPos = std::find(targets.second.begin(), targets.second.end(), targetName);
+            if (targetPos != targets.second.end())
+                return infra::MakeOptional(targets.first);
+        }
+        return infra::none;
+    }
+
+    bool UpgradePackBuilderFacade::CheckIfTargetIsInOrder(const std::string& target, const application::SupportedTargets& supportedTargets)
```

Comment:
"Check..." is a verb; it must do something. Instead, the primary purpose of this function is to decide on a predicate. That should be reflected in the name, so the name should read like a predicate instead of a verb.

```suggestion
    bool UpgradePackBuilderFacade::IsTargetInOrder(const std::string& target, const application::SupportedTargets& supportedTargets)
```

Reply:
Done